tox.ini

```diff
@@ -90,15 +90,26 @@ commands =
     python -c "import subprocess, sys; output = subprocess.check_output('isort -q --diff --recursive deluge docs packaging *.py', shell=True); print output; sys.exit(len(output) != 0)"
 
 [testenv:flake8]
+# Force installing flake8 package or else it will use the system installed if it exists.
+# The flake8 executable has the python path used at install time hardcoded, so if
+# using the system flake8 the wrong libraries are imported.
```

Comment:
I think this would be a little more concise:

"Forcing install of flake8 with pip as system installed flake8 uses a hardcoded python path which can import the wrong libraries"